content/en/docs/concepts/storage/volumes.md

```diff
@@ -58,9 +58,15 @@ Volumes cannot mount within other volumes (but see [Using subPath](#using-subpat
 for a related mechanism). Also, a volume cannot contain a hard link to anything in
 a different volume.
 
-## Types of volumes {#volume-types}
+## In-tree volume plugins and their CSI Migration progress {#volume-types}
+
+Below is a list of in-tree plugins that are supported by Kubernetes and the status of their respective CSI Migrations progress.
+The Container Storage Interface (CSI) was designed to help Kubernetes replace its existing, in-tree storage driver mechanisms - especially vendor specific plugins. Kubernetes support for the Container Storage Interface has been generally available since Kubernetes v1.13. For more details on CSI Migration, please refer to [Out-of-tree volume plugins](#out-of-tree-volume-plugins).
+
+{{< note >}}
+When a Kubernetes cluster administrator updates a cluster to enable CSI migration, existing workloads that utilize PVCs which are backed by in-tree storage plugins will continue to function as they always have. Existing in-tree PVs and in-tree format yaml file will still be used in the future even after the migration is completed, and will not be left unprocessed or deleted in the future. Users can still specify the following in-tree plugins in their yaml file. However, behind the scenes, Kubernetes hands control of all storage management operations (previously targeting in-tree drivers) to CSI drivers.
```

Comment:
Would you consider expressing it more concisely as follows?

If you switch that plugin to an out-of-tree one in the Kubernetes configuration, or if the plugin is removed from the in-tree, the workload can still use the existing manifest yaml file or the PV created without any changes. This behavior is ensured by the migrated out-of-tree plug-in.